### PR TITLE
Updated forward movement

### PR DESCRIPTION
**Pull request by Keigun-Spark**
Added sand and gravel protection

### CODE REVIEW

**Comment by Keigun-Spark** [comment]
Added stripTunnel.lua for two high tunnels with torches
Added api.drop to drop turtle content into chest infront
Added api.avoidChest to turn around if chest infront

**Comment by Keigun-Spark** [comment]
Added api.emptyInv to drop all loot into a chest when called and return back to the mining position.

**Comment by Keigun-Spark** [comment]
Added api.waitforemptyInv function which waits for user input to return to saved position and continue script.

**Comment by Keigun-Spark** [comment]
Code cleanup and auto refuel

**Comment by Keigun-Spark** [comment]
Turned old stripTunnel into mineTunnel
New stripTunnel is now classic strip mining, torch placement not tested yet

**Comment by Keigun-Spark** [comment]
Fixed torch placement in stripTunnel.lua
Changed customAPI functions api.left and api.right

File: apis/storageAPI.lua [hunk 1]
```diff
@@ -0,0 +1,56 @@
+local data = require("dataAPI")
+local move = require("moveAPI")
+
+local storage = {
+	maxSlots = 16,
+}
+
+function storage.drop(slots)
```
Comment (drums12360) [inline apis/storageAPI.lua:8]: there is no use for this function argument

**Review by drums12360 — APPROVED** [review]

**Pull request merged**